src/core/Akka/Actor/Internal/ActorSystemImpl.cs

```diff
@@ -320,12 +322,21 @@ private void LoadExtensions()
 
         private void ConfigureExtensions(IEnumerable<IExtensionId> extensionIdProviders)
         {
-            foreach(var extensionId in extensionIdProviders)
+            foreach (var extensionId in extensionIdProviders)
             {
                 RegisterExtension(extensionId);
             }
         }
 
+        private void ConfigureExtensions(ActorSystemSetup setup)
+        {
+            var extensions = setup.Get<BootstrapSetup>()
+                .Select(_ => _.Extensions)
+                .GetOrElse(ImmutableArray<IExtensionId>.Empty);
+
+            ConfigureExtensions(extensions);
```

Comment:
So `ConfigureExtensions` will be called twice if there are some extensions defined in HOCON _and_ some via the `BootstrapSetup`, correct? And the `RegisterExtension` method is idempotent, so even if the same extension was registered twice there shouldn't be any issues?

Reply:
In the event that there's a HOCON collection for extensions and a `BootstrapSetup`, what happens?